### PR TITLE
Application and configuration

### PR DESCRIPTION
**Pull request by jerith**

### CODE REVIEW

File: config/marathon_event_exporter.schema.exs [hunk 1]
```diff
@@ -0,0 +1,85 @@
+@moduledoc """
+A schema is a keyword list which represents how to map, transform, and validate
```
Comment (JayH5) [inline config/marathon_event_exporter.schema.exs:2]: I'm wondering where all these docs come from... is there a command you ran to generate this file?

**Review by JayH5 — APPROVED** [review]
:+1: so far... How would one set the logging level?

File: config/marathon_event_exporter.schema.exs (hunk 1 above)
Comment (jerith) [reply config/marathon_event_exporter.schema.exs:2]: Yes, https://hexdocs.pm/conform/getting-started.html is the relevant documentation for that.

I edited after it was generated, though.

**Review by JayH5 — APPROVED** [review]
:+1:

**Pull request merged**